frontend/integration-tests-cypress/cypress/support/commands.ts

```diff
@@ -0,0 +1,53 @@
+import { appHost, timeouts } from './utils';
+
```

Comment:
Can we split this up, withina `support/commands` directory to:

```
- /support/commands/selectors.ts
- /support/commands/project.ts
- /support/commands/list-page.ts
- /support/commands/details-page.ts
- /support/commands/nav.ts 
```

I see this file growing exponentially over time, so I'd rather encourage organization up front.

Reply:
It looks like the `support/commands.js` file is loaded first by design:
https://docs.cypress.io/api/cypress-api/custom-commands.html#Syntax
which is fine, so perhaps we have:
```
- /support/commands.ts // this is really acting like an index file
- /support/commands/selectors.ts
- /support/commands/project.ts
- /support/commands/list-page.ts
- /support/commands/details-page.ts
- /support/commands/nav.ts 
```

and let the `commands.ts` file import `commands/project.ts`, etc.  It can compose the subsets.

Reply:
New directory structure is
![image](https://user-images.githubusercontent.com/12733153/80722540-77240700-8acd-11ea-977a-369cd2ff915f.png)
Where 'cy-command-additions' contains everything needed to add to the Cypress global `cy.` variable.  `utils` and `views` are straight up javascript helper classes.